bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/XmlQName.java

```diff
@@ -81,6 +81,19 @@ public XmlQName(BString localName, BString uri, BString prefix) {
         }
     }
 
+    @Deprecated
+    public XmlQName(String qNameStr, String prefix) {
+        int parenEndIndex = qNameStr.indexOf('}');
```

Comment:
1. I think what we need in the `lastIndexOf('}')`, we need to change in line:63 too.
2. We can just to `this(qNameStr); this.prefix = prefix;` right

Reply:
Yes. Will update.